src/lib/adamant-api/index.js

```diff
@@ -129,6 +130,19 @@ export function getPublicKey(address = '') {
   })
 }
 
+/**
+ * Update message in the chat store
+ * @param {{
+ *   id: number,
+ *   realId?: string,
+ *   status: number,
+ *   partnerId: string
+ * }} payload
+ */
+function updateStoreMessage(payload) {
+  store.commit('chat/updateMessage', payload)
+}
```

Comment:
Better not to import Vuex store in adamant-api module. We have a task to get rid of this approach